static/app/views/lastKnownRouteContextProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import {createContext} from 'react';
+
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import usePrevious from 'sentry/utils/usePrevious';
+import {useRoutes} from 'sentry/utils/useRoutes';
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+export const LastKnownRouteContext = createContext<string>('');
```

Comment:
let's start with a default like `<unknown>`

Reply:
would you cast the return value of the hook then?

```
export function useLastKnownRoute() {
  return useContext(LastKnownRouteContext) as string;
}

```

Reply:
Oh sorry I literally mean the string “<unknown>”